Library/Homebrew/install.rb

```diff
@@ -10,13 +10,18 @@ module Install
     module_function
 
     def check_cpu
-      case Hardware::CPU.type
-      when :ppc
+      if OS.mac? && Hardware::CPU.ppc?
```

Comment:
```suggestion
      if Hardware::CPU.ppc?
```

Reply:
I get it, so this is by default for mac. What do you think about moving this to `Library/Homebrew/extend/os/mac/install.rb`?

Reply:
We tend to keep the Mac defaults in `install.rb` and then override for Linux.